app/jobs/pull_updates.rb

```diff
@@ -4,28 +4,10 @@ class PullUpdates < ApplicationJob
   def perform
     last_update = Submission.maximum(:app_store_updated_at) || Time.zone.local(2023, 1, 1)
 
-    json_data = HttpPuller.new.get_all(last_update)
+    json_data = AppStoreClient.new.get_all_submissions(last_update)
 
     json_data['applications'].each do |record|
-      save(record['application_id'], convert_params(record))
+      ReceiveApplicationMetadata.new(record).save
```

Comment:
as per my submit message... I no longer think saving here is the correct idea... this should just create a job to pull all the data in.. :)